src/components/molecules/header/Header.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+//TODO: IMPORT USESTATE
+//TODO: IMPORT LINK FROM ROUTER DOM
+
+//IMPORT COMPONENTS
+import Logo from "@atoms/logo/desktop/DesktopLogo";
+import Link from "@atoms/link/Link";
+import Menu from "@atoms/menu/Menu";
+
+//TODO: IMPORT STYLES
+import {
+  HeaderWrapper,
+  HeaderContainer,
+  HeaderLeftContainer,
+  HeaderContainerRight,
+  Line,
+} from "./Header.styles";
+
+// IMPORT ASSETS
+import { languages } from "@utils";
+
+const Header = () => {
+  // HOOKS
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
```

Comment:
This `useEffect` should be in the `// LIFECYLES` section.

Reply:
done